src/CodeCompliance/Domain/Checks/PrivateApi/Used/PersistenceInBusinessModel.php

```diff
@@ -90,7 +99,9 @@ public function getViolations(): array
                         )
                     ) {
                         $guideline = sprintf($this->getGuideline(), $methodReflection->getDeclaringClass()->getName(), $methodName, $source->getClassName());
-                        $violations[] = new Violation((string)(new Id()), $guideline, $this->getName());
+                        $violations[] = new Violation((string)(new Id()), $guideline, $this->getName(), ViolationInterface::SEVERITY_ERROR, [
+                            'documentation' => $this->getDocumentationUrl(),
```

Comment:
Better to create const for `documentation` key on CodeCompliance level

Reply:
\+